test/status-report-test.coffee

```diff
@@ -0,0 +1,88 @@
+jsonFile = require 'json-file-plus'

```

Comment:
Any reason you arent just using json.parse and fs.openSync() ?  Just seems like overkill for not writing files either